docs/interface.md

```diff
@@ -116,3 +116,34 @@ when a share is sent to a native application, the user agent may create an
 object with `ACTION_SEND`, setting the `EXTRA_SUBJECT` to `title`. Since Android
 intents do not have a URL field, `EXTRA_TEXT` would be set to `text` and `url`
 concatenated together.
+
+## Security considerations
+
+Implementations should observe the following security and privacy advice.
+
+Web Share enables data to be sent from websites to native applications. While
+this ability is not unique to Web Share, it does come with a number of potential
+security issues that may vary in severity (depending on the underlying
+platform).
+
+* Implementations MUST NOT allow the website to learn which apps are
+  installed, or which app was chosen from `navigator.share`. This information
```

Comment:
s/which apps/which native applications

Also, maybe change "Implementations" to use agent, so it's more clear that in which direction the information is moving and who is moving it... saying "Implementations" is a bit vague, IMO.